tests/layout/test_checkbox.py

```diff
@@ -44,3 +44,15 @@ def test_no_help_text_errors(snapshot_html):
     form = CheckboxForm(data={"accept": ""})
     form.fields["accept"].help_text = ""
     assert render_form(form) == snapshot_html
+
+
+def test_html_escaping(snapshot_html):
```

Comment:
The test suite uses `pytest`, so you could use parameterisation to write a single test that checks each field and avoid creating almost identical tests for each field.

https://docs.pytest.org/en/7.1.x/how-to/parametrize.html

Reply:
Thanks for your review! 

I'm a big fan of Pytest's `parametrize`, and use it [very](https://github.com/Textualize/textual/pull/486/files#diff-c426a5671015881709b96fef15705b67301ba7d5cfe8e6d07af2be6050d4c79cR99) [often](https://github.com/Textualize/textual/pull/501/files#diff-cd29d4d0975ff6cbdc45010f04821cf099ac44d1472f0ed4b75e8425d1af6c2dR200) :smile:   

In this case however, what I'm testing is not a `input -> expected output` matrix but just a few assertions on a single input - which I think doesn't fit the use of `parametrize`? :slightly_smiling_face:

Reply:
Sorry, that was rather unclear wasn't it...

I was thinking we could parameterise the test and assert whether escaping happens for several fields and templates instead of just checking `BooleanField` and `checkboxes.html`.

While the test you've added is great, it doesn't prevent us from applying `|safe` to a different template.